core/src/main/java/org/fao/geonet/api/records/attachments/FilesystemStoreResource.java

```diff
@@ -61,23 +72,34 @@ public String getUrl() {
     }
 
     @Override
-    public String getType() {
-        return metadataResourceVisibility.toString();
+    public MetadataResourceVisibility getVisibility() {
```

Comment:
If we rename the getter, this will affect the response.
If we change the type to an enum, this may affect the response too (uppercase now).

Some client side work needed 
![image](https://user-images.githubusercontent.com/1701393/74023888-fdceb980-49a0-11ea-9a8c-0e9c94de9eff.png)